src/plugins/condition/utils/operations.js

```diff
@@ -4,7 +4,7 @@ export const OPERATIONS = [
     {
         name: 'equalTo',
         operation: function (input) {
-            return input[0] === parseInt(input[1]);
+            return Number(input[0]) === Number(input[1]);
```

Comment:
It seems as though one of these casts is unnecessary if the operation only applies to numbers?